packages/volto/src/helpers/Blocks/cloneBlocks.test.js

```diff
@@ -0,0 +1,54 @@
+import { cloneDeep } from 'lodash';
+import { cloneBlocks } from './cloneBlocks';
+
+describe('cloneBlocks', () => {
+  it('cloneBlocks - basic', () => {
+    const blocksData = {
+      blocks: {
+        a: { '@type': 'slate', value: 1 },
+        b: { '@type': 'slate', value: 2 },
+      },
+      blocks_layout: { items: ['a', 'b'] },
+    };
+    const result = cloneBlocks(blocksData);
+
+    expect(result.blocks_layout.items[0]).not.toBe('a');
+    expect(result.blocks[result.blocks_layout.items[0]]).not.toBe({
```

Comment:
Isn't this the assertion we want here?

```suggestion
    expect(result.blocks[result.blocks_layout.items[0]]).toStrictEqual({
```

Reply:
Fixed